exercises/bob/.meta/generator/bob_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class BobCase < Generator::ExerciseCase
+  def workload
+    indent_lines(["remark = %Q(#{input})",
+      "assert_equal '#{expected}', Bob.hey(remark), %Q(Bob hears #{input}, and..)"
```

Comment:
@Insti Adding ```inspect``` here actually causes several tests to close the %Q() call early,because those test cases contain literal ```)``` in the actual values. This causes a syntax error on line 101 of the generated test.

If I do something like ```'Bob hears #{input.inspect}, and..'``` then I get the same issue because certain output contains ```'```. 

I'm basically trapped in a big catch 22 of string interpolation.